samples/sampler/stories/default/Button.js

```diff
@@ -33,22 +35,40 @@ const threeWayBoolean = (value) => {
 	}
 };
 
-const SkinnedButton = Skinnable(
-	{prop: 'skin'},
-	({skin, ...rest}) => {
+const SkinnedButtonBase = kind({
+	name: 'SkinnedButton',
+
+	propTypes: {
+		skin: PropTypes.string
+	},
+
+	render: ({skin, ...rest}) => {
 		let icons = skin === 'silicon' ? ['', ...iconListSilicon] :  ['', ...iconList];
 
 		return (
 			<Button
 				{...rest}
+				icon={select('icon', icons, Config)}
```

Comment:
You're right, `icon` is moved to the bottom because is defined in `SkinnedButton`. I also moved `iconFlip` and `iconPosition` in `SkinnedButton` to group them together with `icon`.
`icon` does not appear in story source because is defined in `SkinnedButton`. We need to find a way to pass it down to `Button`...